opentelemetry-sdk/tests/performance/benchmarks/trace/test_benchmark_trace.py

```diff
@@ -0,0 +1,51 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import opentelemetry.sdk.trace as trace
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import sampling
+
+tracer = trace.TracerProvider(
+    sampler=sampling.DEFAULT_ON,
+    resource=Resource(
+        {
+            "service.name": "A123456789",
+            "service.version": "1.34567890",
+            "service.instance.id": "123ab456-a123-12ab-12ab-12340a1abc12",
+        }
+    ),
+).get_tracer("sdk_tracer_provider")
+
+
+def test_simple_start_span(benchmark):
+    def benchmark_start_as_current_span():
+        span = tracer.start_span(
+            "benchmarkedSpan",
+            attributes={"long.attribute": -10000000001000000000},
+        )
+        span.add_event("benchmarkEvent")
+        span.end()
+
+    benchmark(benchmark_start_as_current_span)
+
+
+def test_simple_start_as_current_span(benchmark):
+    def benchmark_start_as_current_span():
+        with tracer.start_as_current_span(
```

Comment:
I'm not sure if I understand the specification correctly, should this span have an [associated resource](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/performance-benchmark.md#span-configuration)?

Reply:
Yes it already does! You can only create a `Span` through a `Tracer` from the `TracerProvider`. So when I added a resource to the `TracerProvider` [in the code above](https://github.com/open-telemetry/opentelemetry-python/pull/1443/files/a95b89783cf2edfa59e8a6e8b9ec5dcf6d31937e#diff-05e14e7525bcf3fa106e6675e17385c8c2462911c541249b6046c2fb1c9e7dcbR21-R27), every span created through the `Tracer` gets the same Associated Resource! 😄

Reply:
Ah, cool. Approving :+1: